storage/google/cloud/storage/blob.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint: disable=too-many-lines
```

Comment:
Not specifically:  we hit an arbitrary, self-imposed limit.  Breaking the file up isn't necessarily a good thing.  You might cast the issue as "review those limits and the files where we disable them."